### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by szepeviktor**
Found few misspellings.

### CODE REVIEW

**Comment by szepeviktor** [comment]
Some workflows are not PR-friendly.
- cannot deploy GH pages
- Mega Linter cannot comment on the PR

File: tests/fixtures/sniffs/introduced.php
Comment (llaville) [inline tests/fixtures/sniffs/introduced.php:1]: Sorry this is not typo error here

**Review by llaville — COMMENTED** [review]
Thanks for fixing these typos.

Revert changes on tests/fixtures/sniffs/introduced.php and I will accept your PR

File: tests/fixtures/sniffs/introduced.php
Comment (szepeviktor) [reply tests/fixtures/sniffs/introduced.php:1]: I just was unable to read that line.

Comment (llaville) [reply tests/fixtures/sniffs/introduced.php:1]: Revert code changes on the tests/fixtures/sniffs/introduced.php file and I'll accept your PR

previous code is 
```
$a = array('<foo>', "'bar'", '"baz"', '&blong&', "\xc3\xa9");
```

Comment (szepeviktor) [reply tests/fixtures/sniffs/introduced.php:1]: It was done in the previous commit.

**Review by llaville — APPROVED** [review]

**Comment by szepeviktor** [comment]
Glad to contribute.

**Pull request merged**